GameData.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    /// <summary>
+    /// Data of application.
+    /// </summary>
+    public class GameData
+    {
+        readonly public int[,] data;
+        readonly public int[,] newData;
+
+        /// <summary>
+        /// Data of application.
+        /// </summary>
+        public GameData()
+        {
+            int column = UserInterface.GetValueInRange("Input amount of columns", 5, 200);
+            int row = UserInterface.GetValueInRange("Input amount of rows", 5, 200);
+            data = new int[row, column];
+            newData = new int[row, column];
+            FillDefault();
+        }
+
+        /// <summary>
+        /// Fills gameboard with random alive cells.
+        /// </summary>
+        void FillDefault()
```

Comment:
FillDefault what and where?

Reply:
Changed summary and function name to better describe its purpose.